libs/wingsdk/test/fs/fs.test.ts

```diff
@@ -0,0 +1,83 @@
+import { test, expect } from "vitest";
```

Comment:
Can you please add an example written in wing for each of the methods? (here: `examples/tests/sdk_tests/fs`)